api/terraform/data_api/variables.tf

```diff
@@ -2,24 +2,14 @@ variable "aws_region" {
   default = "eu-west-1"
 }
 
-variable "infra_bucket" {}
-
 variable "namespace" {
   default = "data_api"
 }
 
-variable "snapshot_generator_release_uri" {}
-
-variable "critical_slack_webhook" {}
-
-variable "vpc_id" {}
-
-variable "private_subnets" {
-  type = "list"
+variable "critical_slack_webhook" {
+  default = ""
```

Comment:
Critical with empty value?

Reply:
Yeah cos the alarm isn’t currently hooked up. It’s how it was before I started. 🤷‍♀️